Data Science Fundamentals: Python |
[Table of Contents](../index.ipynb)
- - - 
<!--NAVIGATION-->
Module 1. [Basic Python Syntax](./01_mod_python_syntax.ipynb) | [Variables and Objects](./02_python_variables_objects.ipynb) | [Operators](./03_python_operators.ipynb) | **[Exercises](./04_mod_practice.ipynb)**

# Module 1: Practice Exercises
Please remember to submit this file to Titus

### Exercise 1: Accept the user's first and last name and print them in reverse order with a space between them.

In [4]:
#insert your code here, and don't forget to comment EVERY line!

#To get user first name.
firstName = input("Insert first name!:")

#To get user last name.
lastName = input("Insert last name!:")

#Prints last name then first name.
print(lastName, firstName)


Insert first name!:Kahlil
Insert last name!:Cole
Cole Kahlil


### Exercise 2. Accept an integer (n) input from the user and compute the value of n+nn+nnn
For example if `n=4` then `n + nn + nnn = 4 + 44 + 444 = 492`

In [9]:
#insert your code here

#Get n input
n = int(input("Insert number:"))

#Get nn input
nn = int(input("Insert number:"))

#Get nnn input
nnn = int(input("Insert number:"))

#Equation to compute n+nn+nnn
x = n + nn + nnn

print(x)

Insert number:4
Insert number:44
Insert number:444
492


### Exercise 3. Ask the user "What country are you from?" then print the following statement: "I have heard that [input] is a beautiful country!"

In [33]:
#insert your code here

#country = input("What country are you from? ")

#print("I have heard",country,"is a beautiful country!")

print(2 * 3 ** 3 * 4)

216


### Exercise 4. What is the output of the following python code?

#Print your answer here

Nothing because 100 is equal to 100 so you would not get an output because of the and function both parts need to pass. If it was >= then you would get an output.


### Exercise 5: What is the output of the following addition `+` operator, and why does this code chunk execute this way??

#Print your answer here

Since b = a then you would be adding [30, 40] to the list [10, 20] to make the list [10, 20, 30, 40]

### Exercise 6. What is the output of the following code, and what operator is being used here?

#Print your answer here

We are using the mod operator. Since 2 is not fully divisible by 6 we get 2 

### Exercise 7: What is the output of the following code, and what arithmetic operators are used in this code?

#Print your answer here

We are using multiplications (*) and Exponentiation	(**). 

We get 216 as an output because 3^3*4=108 108*2=216.

### Exercise 8: What is a text editor?

#Print your answer here
A text editor is the program you use to write code.

### Exercise 9: What is python?

#Print your answer here

Python is an object oriented programming language.

### Exercise 10: What is jupyter notebook, what type of python environment is it, and what alternatives are there to jupyter notebook?

#Print your answer here
Jupyter Notebook is an open source web app used for programming. An alternate to jupyter notebook would be pycharm, anaconda, vim, and vscode.

### Please remember to submit this file to Titus!

<!--NAVIGATION-->
Module 1. [Basic Python Syntax](./01_mod_python_syntax.ipynb) | [Variables and Objects](./02_python_variables_objects.ipynb) | [Operators](./03_python_operators.ipynb) | **[Practice](./04_mod_practice.ipynb)**
<br>
[top](#)

- - -

Copyright © 2020 Qualex Consulting Services Incorporated.